docs/.vuepress/theme/styles/theme.styl

```diff
@@ -277,5 +145,78 @@ table
     border 1px dotted $colorPrimary
     box-shadow 0 4px 8px $boxShadowHoverColor
 
+// BASE TEXT STYLES
+// These target mainly our markdown content, and whenever a .l# class is not used
+// They should eventually be moved out of utils.styl
+
+h1, h2, h3, h4, h5, h6, p, span, li {
+  max-width: $contentWidth;
+  // This prevents content from being hidden from the fixed header when an anchor tag is used
+  &:target:before {
+    content: "";
+    display: block;
+    height: 60px; /* fixed header height*/
+    margin: -60px 0 0; /* negative fixed header height */
+  }
+}
+
+h1:not(.l1):not(.l2):not(.l3):not(.l4):not(.l5):not(.l6):not(.l7):not(.l8)
+  @extend .l1
+h2:not(.l1):not(.l2):not(.l3):not(.l4):not(.l5):not(.l6):not(.l7):not(.l8)
+  @extend .l2
+h3:not(.l1):not(.l2):not(.l3):not(.l4):not(.l5):not(.l6):not(.l7):not(.l8)
+  @extend .l3
+h4:not(.l1):not(.l2):not(.l3):not(.l4):not(.l5):not(.l6):not(.l7):not(.l8)
+  @extend .l4
+h5:not(.l1):not(.l2):not(.l3):not(.l4):not(.l5):not(.l6):not(.l7):not(.l8)
+  @extend .l5
+h6:not(.l1):not(.l2):not(.l3):not(.l4):not(.l5):not(.l6):not(.l7):not(.l8)
+  @extend .l6
+p:not(.l1):not(.l2):not(.l3):not(.l4):not(.l5):not(.l6):not(.l7):not(.l8)
+  @extend .l7
+  & + p
+    margin-top 2em
+
+p
+  @extend .tc-text300
+  
+.dark-mode p
+  @extend .dark-mode .tc-text300
+
+.content__default
+  ul
+    @extend .l7
+    padding 0
+    margin 1em
+    list-style-type none
+    list-style-image none
+
+  li
+    padding-left .5em
+    margin-bottom .5em
+
+    &:before
```

Comment:
This will create an issue for right-to-left languages, e.g. see https://deploy-preview-801--ethereumorg.netlify.com/ar/ vs. https://ethereum.org/ar/

Reply:
Perhaps you can solve by overwriting when the `.right-to-left-text` class is present

Reply:
This has been resolved with a relatively significant update, all utility classes, list style updates, and the homepage arrows now have x-padding/margin values contained like so:
```
.div:not(.rtl)
  .class
    padding-left: 1rem

.rtl
  .class
    padding-right 1em
```
I changed `.right-to-left-text` to `.rtl`